### PR TITLE
Fix world info in Guided Tour

### LINKED ISSUE #4709: Guided Tour window shows wrong simulation description
The simulation description in the Guided Tour window doesn't match the open world.
Just the numbers are correct.

Screenshot from 2022-06-22 08-19-33

I can reproduce it on Ubuntu 22.04 with the snap installation and on my development system.

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Fix #4709.

If loading a guided tour world takes long (usually because of the assets download) the Guided Tour window is updated before the `WbWorld::instance()` object is updated and thus still shows infos related to the previously loaded world.

While waiting for the world to load, the title label shows the world file name and the info contains the "Loading.." text.
Then, after the world loading is completed and `WbWorld::instance()`  is reset, the title and info are replaced by the actual `WorldInfo` field values.
Screenshot from 2022-06-22 09-45-07

### CODE REVIEW

File: src/webots/gui/WbGuidedTour.cpp [hunk 1]
```diff
@@ -201,10 +203,19 @@ void WbGuidedTour::updateGUI() {
     mInfoText->setPlainText(tr("Thanks for viewing the Webots Guided Tour.") + "\n\n" + tr("Press [Close] to terminate..."));
   } else {  // Normal case
     // Sets world's title
-    setTitleText(WbWorld::instance()->worldInfo()->title() + QString(" (%1/%2)").arg(mIndex + 1).arg(mFilenames.size()));
-    // Formats and displays all WorldInfo.info items
-    const WbMFString &info = WbWorld::instance()->worldInfo()->info();
-    mInfoText->setPlainText(formatInfo(info));
+    if (!WbWorld::instance()->fileName().endsWith(mFilenames[mIndex])) {
+      // New world still loading
+      // Reset title and info until correct info are available
```
Comment (omichel) [inline src/webots/gui/WbGuidedTour.cpp:208]:
```suggestion
      // Reset title and info until correct info is available
```

**Review by omichel — CHANGES_REQUESTED** [review]

**Review by omichel — APPROVED** [review]
That looks better to me this way.
Thank you.

**Comment by stefaniapedrazzi** [comment]
I had to fix a clang-format error.
Build test passed (before I synchronized with develop).

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**